lib/galaxy/util/sanitize_html.py

```diff
@@ -194,7 +194,7 @@ def convert_entityref(self, name):
 
     def output(self):
         '''Return processed HTML as a single string'''
-        return ''.join([str(p) for p in self.pieces])
+        return ''.join([unicodify(p) for p in self.pieces])
```

Comment:
Yeah, if all the pieces are the correct types by default we can just remove the redundant cast/conversion.  Updated it, will test it a bit.